docs/ops.html

```diff
@@ -1890,6 +1890,16 @@ <h4><a id="selector_monitoring" href="#selector_monitoring">Common monitoring me
         <td>The average length of time the I/O thread spent waiting for a socket ready for reads or writes in nanoseconds.</td>
         <td>kafka.[producer|consumer|connect]:type=[producer|consumer|connect]-metrics,client-id=([-.\w]+)</td>
       </tr>
+      <tr>
+        <td>io-wait-time-ns-total</td>
+        <td>The total time the I/O thread spent waiting in nanoseconds</td>
+        <td>kafka.[producer|consumer|connect]:type=[producer|consumer|connect]-metrics,client-id=([-.\w]+)</td>
+      </tr>
+      <tr>
+        <td>io-waittime-total</td>
+        <td><b>*Deprecated*</b> The total time the I/O thread spent waiting in nanoseconds</td>
+        <td>kafka.[producer|consumer|connect]:type=[producer|consumer|connect]-metrics,client-id=([-.\w]+)</td>
+      </tr>
```

Comment:
Added the pairs of metrics to the documentation site. The metric with the wrong name wasn't present in the docs before.